FHICORC/Utils/DateUtils.cs

```diff
@@ -14,7 +14,7 @@ public static string DateToScannerResultFormat(this DateTime date)
 
         public static string ToLocaleDateFormat(this DateTime date)
         {
-            return date.ToString("d", CultureInfo.CreateSpecificCulture(LocaleService.Current.GetLanguage().ToISOCode()));
+            return date.ToString("yyyy-MM-dd", CultureInfo.CreateSpecificCulture(LocaleService.Current.GetLanguage().ToISOCode()));
```

Comment:
This impacts DCC's decoding and presentation, doesn't it? If yes, be sure to test the impact.

Reply:
Yes, but the date is still correct when scanning DCC's